app/views/support_interface/course/open_on_apply.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :browser_title, 'Can candidates use Apply?' %>
```

Comment:
Breadcrumb required also on this page?

Reply:
Following a policy of using back links on pages like this (forms, flows, confirmation steps), and breadcrumbs on other pages where you’re browsing page within a deep hierarchy.